twitter_scraper/modules/profile.py

```diff
@@ -65,6 +67,9 @@ def __parse_profile(self, page):
         if not self.location:
             self.location = None
 
+        self.joined_date = html.find(".ProfileHeaderCard-joinDateText")[0].text.replace('Joined ', '')
```

Comment:
I'm not sure about one thing. Is the element which have **ProfileHeaderCard-joinDateText** class always exists? Because you are not handling the other case.

Reply:
When I check a lot of accounts, it always exists. I couldn't another case.